### PR TITLE
Add PNG `iCCP` tests

### PR DESCRIPTION
**Pull request by svgeesus**
Adding tests for the `iCCP` International Color Consortium profile chunk in PNG.

Tests [developed here](https://github.com/svgeesus/PNG-ICC-tests) and [passed in all browsers](https://svgees.us/PNG/iCCP/results.html).

### CODE REVIEW

**Comment by svgeesus** [comment]
@programmax can you review?

**Comment by svgeesus** [comment]
Oops, css/css-color/parsing/color-valid-rgb.html should not be part of this commit, how do I remove it from the PR without deleting the file?

**Comment by gsnedders** [comment]
> Oops, css/css-color/parsing/color-valid-rgb.html should not be part of this commit, how do I remove it from the PR without deleting the file?

```
git checkout $( git merge-base origin/master HEAD ) -- css/css-color/parsing/
git commit -m 'Revert css/css-color to the original'
```

> Adding tests for the `iCCP` International Color Consortium profile chunk in PNG.
> 
> Tests [developed here](https://github.com/svgeesus/PNG-ICC-tests) and [passed in all browsers](https://svgees.us/PNG/iCCP/results.html).

These currently fail everywhere: https://wpt.fyi/results/png/iCCP?run_id=5169075225886720&run_id=5146773507538944&run_id=5103739344519168

Some of these seem to be anti-aliasing around the edge of the transparent areas, but most are slightly different colours everywhere.

File: png/iCCP/sRGB-untagged.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    }
+    img {
+        padding: 0;
+    }
+    p.test {
+        background-image: url(./support/macbeth-indexed-sRGB-sRGB-chunk.png);
+        background-repeat: no-repeat;
+    }
+</style>
+<h1></h1>
```
Comment (gsnedders) [inline png/iCCP/sRGB-untagged.html:20]: This makes the vertical alignment wrong.

**Review by ProgramMax — APPROVED** [review]
I read through this code review + the README.md in your repo. Conceptually, this all looks good to me.
Once other people's comments are addressed, LGTM on the pull request.

(I'm approving for now, but of course please don't land it until the comments have been addressed.)

**Pull request closed without merging**